{
  "Header": {
    "Original": "CVE-2018-20019 (LibVNC before commit a83439b9fbe0f03c48eb94ed05729cb016f8b72f contains ...)",
    "Line": 314215,
    "ID": "CVE-2018-20019",
    "Description": "(LibVNC before commit a83439b9fbe0f03c48eb94ed05729cb016f8b72f contains ...)"
  },
  "Annotations": [
    {
      "Original": "{DSA-4383-1 DLA-1979-1 DLA-1617-1}",
      "Line": 314216,
      "Type": "xref",
      "Bugs": [
        "DSA-4383-1",
        "DLA-1979-1",
        "DLA-1617-1"
      ]
    },
    {
      "Original": "- libvncserver 0.9.11+dfsg-1.2 (bug #916941)",
      "Line": 314217,
      "Type": "package",
      "Package": "libvncserver",
      "Kind": "fixed",
      "Version": "0.9.11+dfsg-1.2",
      "BugNo": 916941
    },
    {
      "Original": "- italc \u003cremoved\u003e",
      "Line": 314218,
      "Type": "package",
      "Package": "italc",
      "Kind": "removed"
    },
    {
      "Original": "[stretch] - italc 1:3.0.3+dfsg1-1+deb9u1",
      "Line": 314219,
      "Type": "package",
      "Release": "stretch",
      "Package": "italc",
      "Kind": "fixed",
      "Version": "1:3.0.3+dfsg1-1+deb9u1"
    },
    {
      "Original": "NOTE: https://github.com/LibVNC/libvncserver/issues/247",
      "Line": 314220,
      "Type": "NOTE",
      "Description": "https://github.com/LibVNC/libvncserver/issues/247"
    },
    {
      "Original": "NOTE: https://github.com/LibVNC/libvncserver/commit/a83439b9fbe0f03c48eb94ed05729cb016f8b72f",
      "Line": 314221,
      "Type": "NOTE",
      "Description": "https://github.com/LibVNC/libvncserver/commit/a83439b9fbe0f03c48eb94ed05729cb016f8b72f"
    },
    {
      "Original": "NOTE: https://ics-cert.kaspersky.com/advisories/klcert-advisories/2018/12/19/klcert-18-029-libvnc-multiple-heap-out-of-bound-vulnerabilities/",
      "Line": 314222,
      "Type": "NOTE",
      "Description": "https://ics-cert.kaspersky.com/advisories/klcert-advisories/2018/12/19/klcert-18-029-libvnc-multiple-heap-out-of-bound-vulnerabilities/"
    },
    {
      "Original": "NOTE: When fixing this issue apply the complete set of fixes to not open CVE-2018-20748.",
      "Line": 314223,
      "Type": "NOTE",
      "Description": "When fixing this issue apply the complete set of fixes to not open CVE-2018-20748."
    },
    {
      "Original": "NOTE: Additional commits:",
      "Line": 314224,
      "Type": "NOTE",
      "Description": "Additional commits:"
    },
    {
      "Original": "NOTE: https://github.com/LibVNC/libvncserver/commit/c5ba3fee85a7ecbbca1df5ffd46d32b92757bc2a",
      "Line": 314225,
      "Type": "NOTE",
      "Description": "https://github.com/LibVNC/libvncserver/commit/c5ba3fee85a7ecbbca1df5ffd46d32b92757bc2a"
    },
    {
      "Original": "NOTE: https://github.com/LibVNC/libvncserver/commit/e34bcbb759ca5bef85809967a268fdf214c1ad2c",
      "Line": 314226,
      "Type": "NOTE",
      "Description": "https://github.com/LibVNC/libvncserver/commit/e34bcbb759ca5bef85809967a268fdf214c1ad2c"
    },
    {
      "Original": "NOTE: https://github.com/LibVNC/libvncserver/commit/c2c4b81e6cb3b485fb1ec7ba9e7defeb889f6ba7",
      "Line": 314227,
      "Type": "NOTE",
      "Description": "https://github.com/LibVNC/libvncserver/commit/c2c4b81e6cb3b485fb1ec7ba9e7defeb889f6ba7"
    },
    {
      "Original": "NOTE: https://github.com/LibVNC/libvncserver/commit/a64c3b37af9a6c8f8009d7516874b8d266b42bae",
      "Line": 314228,
      "Type": "NOTE",
      "Description": "https://github.com/LibVNC/libvncserver/commit/a64c3b37af9a6c8f8009d7516874b8d266b42bae"
    }
  ]
}